UPGRADE.md

```diff
@@ -1,5 +1,43 @@
 # Upgrade
 
+## dev-develop
+
+### Page-Templates
+
+Page templates will now be filtered: only implemented templates in the theme will be displayed in the dropdown.
+ 
+To find pages with not implemented templates run following command:
+
+```bash
+app/console sulu:content:validate <webspace-key>
+```
+
+To fix that pages, you could implement the template in the theme or save the pages with an other template over ui.
+
+### Webspaces
+
+The excluded xml-tag has been removed from Webspace theme. Remove complete xml-node from all webspace xml files.
```

Comment:
I would write something like (because "Remove complete xml-node" is very ambiguous):

The `excluded` xml tag has been removed from the webspace configuration file, so you have to remove this tag from all these files.